src/container/srv_target.c

```diff
@@ -1663,6 +1678,7 @@ ds_cont_local_open(uuid_t pool_uuid, uuid_t cont_hdl_uuid, uuid_t cont_uuid,
 	dtx_cont_close(hdl->sch_cont, true);
 
 err_cont:
+	hdl->sch_cont->sc_open_initializing = 0;
```

Comment:
This "sc_open_initializing" flag will be cleared when second open return -DER_AGAIN? Why don't we simply use a ABT_Mutex to serialize above open procedures?